submissions/fiyingabriel/css/Product.css

```diff
@@ -0,0 +1,331 @@
+  
+html {
+    display: block;
+}
+body {
+    font-family: Arial, Helvetica, sans-serif;
+}
+/* top navigation */
+.topnav {
+    overflow: hidden;
+    background-color: #004b18;
+    text-align: center;
+    font-size: 17px;
+}
+.topnav a {
+    float: left;
+    padding: 10px 20px;
+    text-decoration: none;
+    color: white;
+}
+/* product page */
+/* top displayed product and detail */
+#shi {
+    float: right;
+    padding-left: 12px;
+}
+#shi li {
+    padding: 0px 0px 10px 10px;
+    font-weight: bold;
+    font-size: 15px;
+}
+.top-prod {
+    height: 350px;
+}
+.top-prod, .detail {
+    display: inline-block;
+    float: left;
+}
+.prod-list {
+    margin-left: 40px;
+    height: 450px;
+}
+.detail {
+    margin-top: 57px;
+}
+#qt {
+    text-align: center;
+    width: 45px;
+    padding: 3px 3px 3px 3px;
+    border-radius: 1px;
+}
+#buy-now {
+    border-radius: 15px;
+    margin-left: 0px;
+    padding: 10px 10px 10px 10px;
+    font-size: 17px;
+}
+/* top display product description */
+.prod-detail {
+    float: left;
+}
+#des {
+    padding-bottom: 1px;
+    font-family: 'Calibri';
+    font-size: 20px;
+}
+/* product images */
+#pi {
+    width: 520px;
+    height: 355px;
+    margin-top: 0px;
+}
+#pc {
+    width: 395px;
+    height: 255px;
+}
+/* other product field */
+#rps {
+    padding-top: 20px;
+    text-align: center;
+}
+.others .three {
+    margin-left: 95px;
+}
+.two, .three, .four {
+    display: inline-block;
+    text-align: center;
+    font-size: 20px;
+    margin-left: 40px;
+    background-color: white;
+    font-weight: bold;
+    padding: 0px 25px 0px 25px;
```

Comment:
This is same as `padding: 0 25px`. You don't need to add a unit to a `0` value in css.